src/client/editor/Editor.js

```diff
@@ -1810,16 +1810,23 @@ export default class Editor {
     await this.exportScene(glbUri, true);
     const { id: glbId, token: glbToken } = await this.project.uploadAndDelete(glbUri);
 
-    const res = await this.project.createOrUpdateScene(
+    const sceneFileUri = this.sceneInfo.uri;
+    const { id: sceneFileId, token: sceneFileToken } = await this.project.uploadAndDelete(sceneFileUri);
```

Comment:
This is kinda dangerous because `uploadAndDelete` will actually delete the scene file off the disk. Usually this is fine because the code in `onPublish` will then immediately save the file, but if `createOrUpdateScene` fails, you will lose your scene file.

Reply:
Ohhh I saw a file disappear while debugging and thought I was crazy!

Reply:
nice catch!